lib/index.js

```diff
@@ -327,6 +331,15 @@ function appendChunk(buf, chunk) {
   }
 }
 
+function applyTxId(event) {
+  if (event && event.request && event.request.headers && event.response && event.response.headers) {
```

Comment:
are all of these checks necessary? We can assume headers is an object and not an instance of some special object.